src/components/popover-surface/popover-surface.tsx

```diff
@@ -44,26 +38,16 @@ export class PopoverSurface {
 
         return (
             <div class={classList} tabindex="-1">
-                <slot name="header" />
                 <slot name="body" />
```

Comment:
Do we still need the name of the slot here? :thinking:

Reply:
I tried removing the named slot but then it will not render anything, I couldn't figure out why so I left it as is.

Reply:
Aha! I guess because then the content you put inside will go to the slot in the slot and that might cause some weird behavior :thinking: We would probably get the same behavior if we do something like this in the parent popover component
```tsx
function render() {
    return <slot slot="body" />
}
```